packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -692,8 +650,8 @@ export default class Editor {
         nameOrMap:
             | string
             | {
-                  [eventName: string]: (event: UIEvent) => void;
-              },
+                [eventName: string]: (event: UIEvent) => void;
+            },

```

Comment:
dat autoformat. we really need prettier or smth on this repo.

Reply:
Not as a pre-commit hook, but I agree. #228 opened.